ingest/validation/validate_metadata.py

```diff
@@ -713,12 +735,15 @@ def value_is_nan(value):
     """
     # We coerces nan to string for group annotations (SCP-2545)
     # string value 'nan' should also be considered value_is_nan=True
-    if value == "nan":
+    if str(value).lower() == "n/a":
```

Comment:
Do we also need to account for `"na"` as well?

Reply:
This is taken care of using the isnan function in line [742](https://github.com/broadinstitute/scp-ingest-pipeline/blob/2bcddddf4ecc6a508be47a198d9751316cbe30c9/ingest/validation/validate_metadata.py#L742).